src/binding/petsc4py/src/petsc4py/PETSc/MatPartitioning.pyx

```diff
@@ -23,38 +24,134 @@ cdef class MatPartitioning(Object):
     def __call__(self):
         return self.getValue()
 
-    def view(self, Viewer viewer=None):
+    def view(self, Viewer viewer=None) -> None:
+        """View the partitioning data structure.
+
+        Collective.
+
+        Parameters
+        ----------
+        viewer
+            A `Viewer` to display the graph.
```

Comment:
or None for the default viewer? @dalcinl We should probably do a final pass and make all `create`, `view`, `setFromoOptions` and `destroy` uniform

Reply:
Yes, we should do a final pass and make it uniform across classes.